msamp/operators/fp8_op/fp8_op.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Fp8Op module."""
```

Comment:
this is actually a dist op instead of fp8 op?

Reply:
I think it is a fp8 op. It just overrides the ncclAllReduce but doesn't provide any dist operator. What's more, current dist_op library is a pytorch extension. What we want is a pure library only depends on nccl.

Reply:
The issue is current filename or class name "FP8Op" or function names "enable_fp8_*" / "disable_fp8"are all general fp8.
It's hard for users to know these are for nccl or torch.dist, sounds like the class is enabling fp8 computation kernels.

Reply:
Make sense. What is your suggestion?

Reply:
I got your point. We should merge dist_op and fp8_op. Let's make it as a TO DO for next version.